Source/Core/DolphinQt/Settings/PathPane.cpp

```diff
@@ -72,6 +72,29 @@ void PathPane::BrowseDump()
   }
 }
 
+void PathPane::BrowseLoad()
+{
+  QString dir = QDir::toNativeSeparators(QFileDialog::getExistingDirectory(
+      this, tr("Select Load Path"), QString::fromStdString(Config::Get(Config::MAIN_LOAD_PATH))));
```

Comment:
"Load Path" seems a bit generic and nondescript to me (same goes for the setting names etc, btw), I don't suppose we have any more useful names for those?
"Resource pack path" on the other hand clearly tells me what it is.

Reply:
Having a tooltip or something as an explanation would be ideal.  The "Load" path is for loading any data (today it is just textures but I already have another review that has different data).  I wasn't sure how to summarize it.  Maybe "LoadData"?

Reply:
No idea. If I had a suggestion, I'd have included it already.